src/components/ux-dropdowns/ux-dropdowns.steps.js

```diff
@@ -0,0 +1,51 @@
+module.exports = function () {
```

Comment:
This entire file can be rewritten as such:

module.exports = function () {

	// Load standard world object to be 'this' in steps.
	this.World = require('../../world').World;
	require('../../support/steps').call(this);


	this.Before(function (callback) {
		this.component = {};
		this.component.container = '#childComponent ';
		this.component.button = this.component.container + ' .button';

		callback();
	});
};

This will reuse the test cases listed in /src/support/steps.js, instead of rewriting them all here.